PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,12 +18,13 @@ Fixes #(issue_number)
 - [ ] Traffic Vault
 - [ ] Other _________
 
-#### What is the best way to verify this PR?
+#### What is the best way to verify this PR? Please include steps or tests. 
```

Comment:
What about changing this to "Please include manual steps or automated tests."

Reply:
Thanks - I updated to reflect this feedback.